sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionCatalog.scala

```diff
@@ -142,8 +142,6 @@ private[sql] class HiveSessionCatalog(
           // let's try to load it as a Hive's built-in function.
           // Hive is case insensitive.
           val functionName = funcName.unquotedString.toLowerCase(Locale.ROOT)
-          logWarning("Encountered a failure during looking up function:" +
-            s" ${Utils.exceptionString(error)}")
```

Comment:
We should not warn here.

Reply:
maybe this is intended for the true case of `if (super.functionExists(name)) ` above? like L138?

Reply:
Ya. Maybe, but I'm not sure about the original intention. For that line, since we already `throw error`, actually, we don't need print warning here with the same `error`.